### PR TITLE
Openapi update

### PR DESCRIPTION
**Pull request by swiftone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -770,6 +771,14 @@ const client = new okta.Client({
 })
 ```
 
+## Migrating between versions
+
+### From 3.x to 4.0
+
+The version 4.0 of this SDK dropped support for Node 8, which is EOL (End-of-Life) since 2019-12-31. Current supported minimum Node version is 10.0.0.
+
+This version 4.0 release also updated APIs latest `@okta/openapi` (v2.0.0) that includes new endpoints and changed method names. Please see [CHANGELOG.md](CHANGELOG.md#400) for detailed information.
```
Comment (bretterer) [inline README.md:780]: You may want to expand on this with how to handle some of the changes that were done. What you have now doesn't help guide the user through the changes, it just shows a diff

File: src/default-request-executor.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018-2020present, Okta, Inc. and/or its affiliates. All rights reserved.
```
Comment (justinabrokwah-okta) [inline src/default-request-executor.js:2]: Just 2020 here

Comment (shuowu) [reply src/default-request-executor.js:2]: good catch!

File: README.md (hunk 1 above)
Comment (justinabrokwah-okta) [reply README.md:780]: +1

**Comment by shuowu** [comment]
> Our copyright should be `2017 - Present` instead of an ending year.

This is a good catch, which means I need to re-generate all banners 👍

**Comment by shuowu** [comment]
Looks like there is a script in the repo to always update banners to current year, I think we should be fine to use 2017-2020 here.

**Pull request merged**